charts/rancher-monitoring/v0.1.3/values.yaml

```diff
@@ -3,10 +3,10 @@ operator:
   enabledRBAC: true
   apiGroup: "monitoring.coreos.com"
   image:
-    repository: rancher/coreos-prometheus-operator
+    repository: quay.io/coreos/prometheus-operator
     tag: v0.39.0
     prometheusConfigReloader:
-      repository: rancher/coreos-prometheus-config-reloader
+      repository: quay.io/coreos/prometheus-config-reloader
```

Comment:
This will be dropped on the actual merge, but I need this to test locally till the images are created